docs/database-restore.md

```diff
@@ -0,0 +1,49 @@
+1. Click the OK button to start the restore process. This will take in the region of ten to fifteen minutes to completely retore the database to a new server, depending upon the size of the database.
+1. While the restore is running, determine the public IP address of your local machine.
+1. Return to the `s106p01-apply` resource group page and select the `s106p01-apply-psql` server from the resource list.
+1. Select "Connection Security" from the Settings menu on the left.
+1. Create a new rule to allow connections from your public IP. Ensure that the rule has a suitable name so that attributed to the user. Once the details have been entered, click Save.
+1. Once the database restore process has finishe go back to the resource group listing and set up a connection rule in the same manner on the restored Postgres server, `s106p01-apply-psql-restored`.
+1. On your local machine run the command `pg_dump -Fc -v -h s106p01-apply-psql-restored.postgres.database.azure.com -p 5432 -U applyadm512@s106p01-apply-psql-restored -W apply -f <output_filname>.dump` to dump the database to your local machine. The Azure postgres servers require SSL connections so you may need to specify the environment variable `PGSSLMODE=required` to establish a connection. The password can be found by examining the app service environment variables in the Azure portal.
+1. Restore the database to the original instance using the command `pg_restore -v --no-owner -h s106p01-apply-psql.postgres.database.azure.com -p 5432 -U applyadm512@s106p01-apply-psql -W -d apply <input_filenme>.dump`
+1. Verify that the database restore has been successful.
+1. Return to the `s106p01-apply` resource group page and select checkbox alongside the restored database instance `s106p01-apply-psql-restored` and click the "Delete" button from the ribon at the top of the resource group blade. In the popup menu that appears confirm the delete operation as directed.
+1. Finally, select the `s106p01-apply-psql` from the resource list and go to "Connection Security" and remove the IP address you added earlier to ensure the server is fully secure again.
+
+
+## PIM Rights Elevation
+
+1. Launch the Azure portal at https://portal.azure.com.
+1. Ensure you are using the "DfE Platform Identity" directory (see step 5 above if not).
+1. In the search bar at the top of the portal type in "PIM" and select "Azure AD Privileged Identity Management" from the search results.
+1. In the new blade that opens select "My roles" from the tasks section of the menu bar on the left.
+1. Under the "Activate" section of the menu select "Azure Resources".
+1. Chose the subscription you want to elevate your rights on and click "Activate". This will launch a new panel where you can specify duration and reason which must be completed before you can click Activate.
```

Comment:
```suggestion
1. Choose the subscription you want to elevate your rights on and click "Activate". This will launch a new panel where you can specify duration and reason which must be completed before you can click Activate.
```